src/GraphQL/Utilities/TypeConfig.cs

```diff
@@ -44,53 +58,11 @@ public void IsTypeOf<T>()
             IsTypeOfFunc = obj => obj?.GetType().IsAssignableFrom(typeof(T)) ?? false;
         }
 
-        public FieldConfig FieldFor(string field, IServiceProvider serviceProvider)
-        {
-            var config = _fields[field];
-            config.ResolverAccessor = Type.ToAccessor(field, ResolverType.Resolver);
-
-            if (Type != null && config.ResolverAccessor != null)
-            {
-                config.Resolver = new AccessorFieldResolver(config.ResolverAccessor, serviceProvider);
-                var attrs = config.ResolverAccessor.GetAttributes<GraphQLAttribute>();
-                if (attrs != null)
-                {
-                    foreach (var a in attrs)
-                        a.Modify(config);
-                }
-            }
-
-            return config;
-        }
-
-        public FieldConfig SubscriptionFieldFor(string field, IServiceProvider serviceProvider)
-        {
-            var config = _fields[field];
-            config.ResolverAccessor = Type.ToAccessor(field, ResolverType.Resolver);
-            config.SubscriberAccessor = Type.ToAccessor(field, ResolverType.Subscriber);
-
-            if (Type != null && config.ResolverAccessor != null && config.SubscriberAccessor != null)
-            {
-                config.Resolver = new AccessorFieldResolver(config.ResolverAccessor, serviceProvider);
-                var attrs = config.ResolverAccessor.GetAttributes<GraphQLAttribute>();
-                if (attrs != null)
-                {
-                    foreach (var a in attrs)
-                        a.Modify(config);
-                }
-
-                if (config.SubscriberAccessor.MethodInfo.ReturnType.GetGenericTypeDefinition() == typeof(Task<>))
-                {
-                    config.AsyncSubscriber = new AsyncEventStreamResolver(config.SubscriberAccessor, serviceProvider);
-                }
-                else
-                {
-                    config.Subscriber = new EventStreamResolver(config.SubscriberAccessor, serviceProvider);
-                }
-            }
-
-            return config;
-        }
```

Comment:
Code was moved into `SchemaBuilder`. Note that `IServiceProvider` parameter was removed from public API here. It is absolutely not needed here.